concent_api/core/views.py

```diff
@@ -56,9 +55,9 @@ def send(request, message):
         current_time = int(datetime.datetime.now().timestamp())
         if current_time <= message.task_to_compute.compute_task_def['deadline'] + settings.CONCENT_MESSAGING_TIME:
 
-            force_task_to_compute   = Message.objects.filter(task_id = message.task_to_compute.compute_task_def['task_id'], type = "ForceReportComputedTask")
-            previous_ack_message    = Message.objects.filter(task_id = message.task_to_compute.compute_task_def['task_id'], type = "AckReportComputedTask")
-            reject_message          = Message.objects.filter(task_id = message.task_to_compute.compute_task_def['task_id'], type = "RejectReportComputedTask")
+            force_task_to_compute   = Message.objects.filter(task_id = message.task_to_compute.compute_task_def['task_id'], type = ForceReportComputedTask().TYPE)
```

Comment:
@kbeker One more thing, you don't have to create instance of a class here, it will work if you do
`ForceReportComputedTask.TYPE` instead of `ForceReportComputedTask().TYPE`.